### PR TITLE
Implement Display for num_bigint::Sign

### PR DESCRIPTION
**Pull request by FliegendeWurst**

### CODE REVIEW

File: bigint/src/bigint.rs [hunk 1]
```diff
@@ -101,6 +101,17 @@ impl serde::Deserialize for Sign {
...
 }
 
+impl fmt::Display for Sign {
+	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		use Sign::*;
+		match *self {
+			Plus => write!(f, "+"),
+			NoSign => write!(f, ""),
+			Minus => write!(f, "-")
+		}
```
Comment (cuviper) [inline bigint/src/bigint.rs]: I think using [`Formatter::pad(&mut self, s: &str)`](https://doc.rust-lang.org/stable/std/fmt/struct.Formatter.html#method.pad) will be more flexible than `write!`.

Comment (FliegendeWurst) [reply bigint/src/bigint.rs]: OK :)

**Comment by FliegendeWurst** [comment]
Here's an [example](https://play.rust-lang.org/?gist=a8b80c748f86400f25ffde35dfba3ba7).
Tests have been added

**Comment by cuviper** [comment]
Did you know you can get almost the same effect with "{:+}" format?  Except that a zero will be printed as "+0", just like primitive integers do, unlike what you've done for `NoSign`.

So your example shows *how* to use it, but I'm looking for *why* you need it.

**Comment by FliegendeWurst** [comment]
Oh I don't need it in that case, thanks (you may close this PR if you want)

**Comment by cuviper** [comment]
OK, thanks anyway!

**Pull request closed without merging**